src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,11 @@
 package mate.academy.lib;
 
-public @interface Inject {
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Inject {
```

Comment:
The `Inject` annotation is defined correctly with both `@Target` and `@Retention` specified as required by the checklist.